### ISSUE TITLE
Several functions result in an error for all arguments

### ISSUE THREAD

**Issue by HolyMeekrob** [transitive-bullshit/random#4]
```javascript
bernoulli()
binomial()
geometric()
irwinHall()
bates()
pareto()
```
all throw errors for all inputs, including defaults.

For example:
```javascript
// Running this
const random = require('random');
const foo = random.bernoulli(0.4);
foo();

// results in
Error: invalid number of arguments to "ow"
    at Object.apply (C:\Users\Andy.Steinberg\Code\test\node_modules\ow-lite\index.js:95:15)
    at exports.default (C:\Users\Andy.Steinberg\Code\test\node_modules\random\dist\distributions\bernoulli.js:16:52)
    at Random.bernoulli (C:\Users\Andy.Steinberg\Code\test\node_modules\random\dist\random.js:369:38)
    at Object.<anonymous> (C:\Users\Andy.Steinberg\Code\test\tester.js:3:20)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:742:12)
```

All of the listed functions result in the same error.

**Comment by zemlyansky** [comment]
It seems that the problem is that part:

binomial.js
```
import ow from 'ow-lite'                                                                                                   
export default (random, n = 1, p = 0.5) => {
ow(n, ow.number.positive.integer)
ow(p, ow.number**.greaterThanOrEqual(0).lessThanOrEqual(1)**)
```

if you remove `.greaterThanOrEqual(0).lessThanOrEqual(1)` everything works fine. Maybe it's a bug of ow-lite

**Comment by bluelovers** [comment]
`ow-lite` api not same as `ow`
so should use `.gte`, `.lte`

https://github.com/bluelovers/random/blob/5aebd0d8458f02c8717308f09c238aaea633ddab/src/distributions/bernoulli.ts#L5-L8

```ts
export default (random: Random, p = 0.5) =>
{
	ow(p, ow.number.gte(0).lte(1))

```

**Comment by transitive-bullshit** [comment]
Sorry for this issue everyone. I've just published a fix in `v2.0.13`.

I've also added unit tests for each distribution to prevent against future regressions.
